packages/web-components/src/message-bar/message-bar.stories.ts

```diff
@@ -0,0 +1,195 @@
+  </fluent-message-bar>
+  <br />
+  <fluent-message-bar shape="square">
+    <span slot="icon">${infoIcon}</span>
+    square
+    <div class="actions" slot="actions">
+      <fluent-button size="small">Action</fluent-button>
+    </div>
+    <fluent-button size="small" appearance="transparent" icon-only slot="close"> ${dismissed20Regular} </fluent-button>
+  </fluent-message-bar>
+`);
+
+export const Layout = renderComponent(html<MessageBarStoryArgs>`
+  <fluent-message-bar layout="singleline">
+    <span slot="icon">${infoIcon}</span>
+    singleline
+    <div class="actions" slot="actions">
+      <fluent-button size="small">Action</fluent-button>
+    </div>
+    <fluent-button size="small" appearance="transparent" icon-only slot="close"> ${dismissed20Regular} </fluent-button>
+  </fluent-message-bar>
+  <br />
+  <fluent-message-bar layout="multiline">
+    <span slot="icon">${infoIcon}</span>
+    This is a message bar that provides information to the user. This is a message bar that provides information to the
+    user.
+    <div class="actions" slot="actions">
+      <fluent-button size="small">Action</fluent-button>
+    </div>
+    <fluent-button size="small" appearance="transparent" icon-only slot="close"> ${dismissed20Regular} </fluent-button>
+  </fluent-message-bar>
+`);
+
+export const Intent = renderComponent(html<MessageBarStoryArgs>`
+  <fluent-message-bar intent="info">
+    <span slot="icon">${infoIcon}</span>
+    info
+    <div class="actions" slot="actions">
+      <fluent-button size="small">Action</fluent-button>
+    </div>
+    <fluent-button size="small" appearance="transparent" icon-only slot="close"> ${dismissed20Regular} </fluent-button>
+  </fluent-message-bar>
+  <br />
+  <fluent-message-bar intent="warning">
+    <span slot="icon">${warningIcon}</span>
+    warning
+    <div class="actions" slot="actions">
+      <fluent-button size="small">Action</fluent-button>
+    </div>
+    <fluent-button size="small" appearance="transparent" icon-only slot="close"> ${dismissed20Regular} </fluent-button>
+  </fluent-message-bar>
+  <br />
+  <fluent-message-bar intent="success">
+    <span slot="icon">${successIcon}</span>
+    success
+    <div class="actions" slot="actions">
+      <fluent-button size="small">Action</fluent-button>
+    </div>
+    <fluent-button size="small" appearance="transparent" icon-only slot="close"> ${dismissed20Regular} </fluent-button>
+  </fluent-message-bar>
+  <br />
+  <fluent-message-bar intent="error">
+    <span slot="icon">${errorIcon}</span>
+    error
+    <div class="actions" slot="actions">
+      <fluent-button size="small">Action</fluent-button>
+    </div>
+    <fluent-button size="small" appearance="transparent" icon-only slot="close"> ${dismissed20Regular} </fluent-button>
+  </fluent-message-bar>
+`);
+
+export const Assertive = renderComponent(html<MessageBarStoryArgs>`
```

Comment:
Great addition to a story! Do we need to add `aria-labelledby` to the examples for a11y purposes? (Asking)